src/test-utils.tsx

```diff
@@ -0,0 +1,44 @@
+import { Router, Route } from 'react-router-dom';
+import React from 'react';
+import fetch from 'node-fetch';
+import { createMemoryHistory, MemoryHistory } from 'history';
+import { render, RenderResult } from '@testing-library/react';
+import { server } from './mocks';
+
+Object.defineProperty(global, 'fetch', {
```

Comment:
We don't need this because we should be using axios

Reply:
Removed